src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetChangeOfProviderChain/GetChangeOfProviderChainQueryResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.CommitmentsV2.Application.Queries.GetChangeOfProviderChain
+{
+    public class GetChangeOfProviderChainQueryResult
+    {
+        public IReadOnlyCollection<ChangeOfProviderLink> ChangeOfProviderChain { get; set; }
+
+        public class ChangeOfProviderLink
```

Comment:
nested classes

Reply:
This is a common pattern in the project - used in at least 5 other places.